temoto_frames:
  human_input: "current_cmd_frame"
  base_frame: "base_link"

temoto:
  temoto_pose_cmd_topic: "/temoto/spacenav/joy"

   # navigate or manipulate
  navigate: false
  manipulate: true

  manip_stl: "package://temoto_support/meshes/2_finger_140_gripper_marker.stl" #"package://temoto_support/meshes/manip_cube.stl"
  input_voice_topic: "stt/spoken_text"
  movegroup: "manipulator"